frontend/lib/search/filter_bar.dart

```diff
@@ -15,8 +15,13 @@ class FilterBar extends StatelessWidget {
   Widget build(BuildContext context) {
     final t = context.t;
     final sortedCategories = [...categoryAssets(context)];
-    sortedCategories.removeWhere((category) => category.id == 9);
     sortedCategories.sort((a, b) => a.shortName.length.compareTo(b.shortName.length));
+
+    // Move the "Others" category to the end of the list
```

Comment:
I assumed we'd want to display it as the last one

Reply:
i prefer using here a named function like `moveOtherCategoryToEnd` or sth similar and do it there so there is no need for a comment. Not sure if adding and removing is the best way
In general we could think of only sorting all elements except of `otherCategoryId` and then add it
```js
 final sortedCategories = [...categoryAssets(context).where((category) => category.id != "otherCategoryId")];
 // then do the sorting and then add the category
 // then add the particular category
    sortedCategories.add(othersCategory);
 
```

Reply:
Applied your suggestion of sorting all except 9 + adding 9